tests/components/hassio/test_addon_manager.py

```diff
@@ -746,9 +751,13 @@ async def test_schedule_update_addon(
 
 @pytest.mark.parametrize(
     (
-        "create_backup_error, create_backup_calls, "
-        "update_addon_error, update_addon_calls, "
-        "error_message"
+        (
+            "create_backup_error",
+            "create_backup_calls",
+            "update_addon_error",
+            "update_addon_calls",
+            "error_message",
+        )
```

Comment:
Should these have double parenthesis?

Reply:
Is has no effect, but reported it upstream.

https://github.com/charliermarsh/ruff/issues/2918

Will check and address existing issues.

Reply:
Soft reset, did an interactive staging to find all cases. Seems like you find them all 👍 

Removed the unneeded parenthesizes.